app/presenters/user_presenter.rb

```diff
@@ -15,4 +14,12 @@ def short_name
   def full_name
     "#{first_name} #{last_name}"
   end
+
+  def gravatar_id
+    Digest::MD5::hexdigest(item.email.downcase)
```

Comment:
Because `#email` is not defined as a method on the presenter, and methods delegate to the item implicitly, you can get away with referencing this as `...hexdigest(email.downcase)`.